syntax = "proto3";

package imagePrep;

message Image {
    bytes gPicture = 1;
}
